packages/tools/wrapError.js

```diff
@@ -105,10 +106,7 @@ export function executingInJest () {
 }
 
 export function inBrowser () {
-  // TODO(iridian): Properly differentiate between browser and jest environments.
-  return window
-      && (true /* !process || (process.env.NODE_ENV !== "production") */)
-      && !executingInJest();
+  return isInBrowser;
```

Comment:
Let's remove the dependency to is-in-browser lib altogether: grab the code, expand it here and remove the is-in-browser dependency:

https://github.com/tuxsudo/is-in-browser/blob/master/src/index.js

The license of that lib is MIT, which allows free rights to copy, modify, merge, etc. content as long as MIT license is included.
If we were not MIT ourselves we would have to think more about how cross-license copying would be allowed, but in this case we're directly compatible.